src/forms/login-popup/index.jsx

```diff
@@ -84,20 +84,16 @@ const LoginForm = () => {
 
   useEffect(() => {
     if (thirdPartyAuthApiStatus === COMPLETE_STATE) {
-      if (providers.length > 0) {
+      if (providers.length > 0 && socialAuthnButtonRef.current) {
         socialAuthnButtonRef.current.focus();
+      } else if (thirdPartyAuthRef.current) {
+        thirdPartyAuthRef.current.focus();
       } else if (emailOrUsernameRef.current) {
         emailOrUsernameRef.current.focus();
       }
     }
   }, [thirdPartyAuthApiStatus, providers]);
 
-  useEffect(() => {
```

Comment:
Why we have to remove it btw?

Reply:
Now we're not explicitly setting the focus on the error alerts, which is why I've removed this `useEffect`.